Sources/NIOHTTP1/HTTPDecoder.swift

```diff
@@ -634,7 +634,7 @@ extension HTTPResponseStatus {
     /// - Parameter statusCode: The integer value of the HTTP response status code
     /// - Parameter reasonPhrase: The textual reason phrase from the response.
     /// - Returns: The corresponding `HTTPResponseStatus`.
-    static func from(_ statusCode: UInt32, _ reasonPhrase: String) -> HTTPResponseStatus {
+    public static func from(_ statusCode: UInt32, _ reasonPhrase: String) -> HTTPResponseStatus {
```

Comment:
@weissi @Lukasa I think we want to  make `statusCode` an `Int` if we make it public to follow what swift usually prefers, right ?

Reply:
yes, should be an `Int` and the whole method should probably be an `init`

Reply:
@weissi +1